doc/jsk_pcl_ros/nodes/container_occupancy_detector.md

```diff
@@ -0,0 +1,36 @@
+# ContainerOccupancyDetector
+## What Is This
+![](images/container_occupancy_detector_rviz.png)
+
+It subscribes containers' boxes (`jsk_recognition_msgs/BoundingBoxArray`) and point cloud in the each boxes (`sensor_msgs/PointCloud2`, `jsk_recognition_msgs/ClusterPointIndices`) then publish their occupancies (`jsk_recognition_msgs/BoundingBoxArray`). Their occupancy rates are in each boxes' `value` field.
+
+## Subscribing Topic
+* `~container/boxes` (`jsk_recognition_msgs/BoundingBoxArray`)
+
+Containers input.
+
+* `~container/points` (`sensor_msgs/PointCloud2`)
+
+Original pointcloud.
+
+* `~container/point_indices` (`jsk_recognition_msgs/ClusterPointIndices`)
+
+The indices of point cloud in each box.
+
+## Publishing Topic
+* `~container/occupancies` (`jsk_recognition_msgs/BoundingBoxArray`)
+
```

Comment:
It's good to add `Parameters` section for `queue_size` and `approximate_sync`.
ref: https://github.com/jsk-ros-pkg/jsk_recognition/blob/master/doc/jsk_perception/nodes/apply_mask_image.md#parameters

Reply:
added